# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: layout/style/FontFaceSetImpl.cpp
# Commit: f724c21b1f15
# Full Hash: f724c21b1f150f300f18e9202a2a0ab2efc4ec57
# Author: Andrew Osmond <aosmond@mozilla.com>
# Date: 2022-06-29 03:44:30
# Regressor Bug: 1771493
# File Overlap Count: 1
# Description:
#   Bug 1771493 - Part 3. Abstract away buffer extraction from FontFaceImpl::InitializeSource. r=emilio
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D147818
# ==============================================================================

diff -r 5a39eb924188 -r f724c21b1f15 layout/style/FontFaceSetImpl.cpp
--- a/layout/style/FontFaceSetImpl.cpp	Tue Jun 28 21:47:00 2022 +0000
+++ b/layout/style/FontFaceSetImpl.cpp	Tue Jun 28 21:47:00 2022 +0000
@@ -880,7 +880,7 @@
     }
 
     face->mSourceType = gfxFontFaceSrc::eSourceType_Buffer;
-    face->mBuffer = aFontFace->CreateBufferSource();
+    face->mBuffer = aFontFace->TakeBufferSource();
   } else {
     AutoTArray<StyleFontFaceSourceListComponent, 8> sourceListComponents;
     aFontFace->GetSources(sourceListComponents);
